### DISCUSSION TITLE
Supervisor errori

### DISCUSSION THREAD

**Discussion by gdemattia1974** [indomus/forum#1317] [GENERALE - Discussioni generiche]
Salve a tutti, ho appena installato HA nella versione OS su Rasphberry pi4.
Ma già al primo avvio, dopo la configurazione inziale il supervisor non viene caricato e andando nel log ho quest'errore:

`ERROR (MainThread) [supervisor.utils] Can't execute check_trust while a task is in progress`

e più sotto anche

`ERROR (MainThread) [supervisor.api.ingress] Ingress error: Cannot connect to host 172.30.32.1:65461 ssl:default [Connect call failed ('172.30.32.1', 65461)]`

`ERROR (MainThread) [supervisor.api.ingress] Ingress error: Cannot connect to host 172.30.32.1:65461 ssl:default [Connect call failed ('172.30.32.1', 65461)]`

sapete darmi qualche info più dettagliata? e soluzione su come avere il supervisror?

**Comment by KC-inDomus** [comment]
ciao, per favore prima adegua il tuo intervento [a queste indicazioni](https://github.com/indomus/forum/discussions/1311).

**Comment by gdemattia1974** [comment]
Chiedo scusa ho avuto troppa fretta nel postare.
